doc/source/conf.py

```diff
@@ -38,6 +47,10 @@
     "github_url": "https://github.com/ansys/ansys-sphinx-theme",
     "use_edit_page_button": True,
     "contact_mail": "pyansys.support@ansys.com",
+    "switcher": {
+        "json_url": "https://raw.githubusercontent.com/ansys/ansys-sphinx-theme/gh-pages/release/versions.json",  # noqa: E501
```

Comment:
Let us use here a variable for the CNAME:

```suggestion
        "json_url": f"https://{cname}/release/versions.json",
```